### PR TITLE
Rancher 1.0.0

### PR DESCRIPTION
**Pull request by jhcook**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/rancher.rb [hunk 1]
```diff
@@ -42,14 +41,14 @@
...
-    "~/Library/Preferences/ByHost/io.rancherdesktop.app*",
-    "~/Library/Preferences/io.rancherdesktop.app.plist",
     "~/Library/Preferences/rancher-desktop",
+    "~/Library/Logs/rancher-desktop",
+    "~/Library/Caches/rancher-desktop",
     "~/Library/Saved Application State/io.rancherdesktop.app.savedState",
+    "~/Library/Preferences/io.rancherdesktop.app.plist",
+    "~/Library/Application Support/Caches/rancher-desktop-updater",
+    "~/Library/Caches/io.rancherdesktop.app*",
+    "~/Library/Preferences/ByHost/io.rancherdesktop.app*",
```
Comment (commitay) [inline Casks/rancher.rb:52]: Why are there being reordered?

Comment (jhcook) [reply Casks/rancher.rb:52]: I grouped the directories as there's comments in Rancher forums about specific directories that need added in this version.

Comment (commitay) [reply Casks/rancher.rb:52]: > comments in Rancher forums about specific directories

Link?

> need added in this version.

But you haven't added any, just rearranged them?

Comment (jhcook) [reply Casks/rancher.rb:52]: Yes I rearranged them and went through the list as well as searched for others. 
https://community.suse.com/posts/clean-install-rancher-desktop

Comment (commitay) [reply Casks/rancher.rb:52]: > Yes I rearranged them

IIRC it's preferred to keep `zap` sorted alphabetically.

> https://community.suse.com/posts/clean-install-rancher-desktop

It's behind a login ...

> as well as searched for others.

Need to add `~/Library/Application Support/Rancher Desktop`

**Pull request closed without merging**